metricbeat/module/elasticsearch/_meta/fields.yml

```diff
@@ -279,6 +394,7 @@
     - name: elasticsearch
       type: group
       description: >
+        TODO
```

Comment:
I think you can remove this `TODO` because it's for a group field. I think that would be better than having `TODO` show up in the docs for this group.

Reply:
Thank you for spotting that TODO :)